scripts/suite.py

```diff
@@ -34,7 +34,6 @@
                               The suite to schedule
   --wait                      Block until the suite is finished
   -c <ceph>, --ceph <ceph>    The ceph branch to run against
-                              [default: {default_ceph_branch}]
```

Comment:
Could you please remind me why this line has been deleted? I thought, there should be just added extra comment, that in case using `--rerun` the branch is taken from the referred run instead.

Reply:
We had TEUTH_CEPH_BRANCH environment variable which default the --ceph when it is not provided. Still unsure if we want to keep this functionality.